Source/WebGPU/WebGPU.xcodeproj/project.pbxproj

```diff
@@ -150,6 +151,24 @@
 		DD05A35C27BF09C60096EFAB /* libWTF.a in Product Dependencies */ = {isa = PBXBuildFile; fileRef = 1CEBD8292716CAE700A5254D /* libWTF.a */; };
 /* End PBXBuildFile section */
 
+/* Begin PBXBuildRule section */
+		9729675F299BF1BE001C8BD4 /* PBXBuildRule */ = {
+			isa = PBXBuildRule;
+			compilerSpec = com.apple.compilers.proxy.script;
+			filePatterns = "*.js";
+			fileType = pattern.proxy;
+			inputFiles = (
+				"$(SRCROOT)/WGSL/generator/main.js",
+			);
+			isEditable = 1;
+			outputFiles = (
+				"$(BUILT_PRODUCTS_DIR)/DerivedSources/WGSL/TypeDeclarations.h",
+			);
+			runOncePerArchitecture = 0;
+			script = "/System/Library/Frameworks/JavaScriptCore.framework/Versions/Current/Helpers/jsc \"${SCRIPT_INPUT_FILE_0}\" -- \"${INPUT_FILE_PATH}\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
```

Comment:
Is this going to work on customer machines?

Reply:
Yes, it passes EWS, which is technically a customer install (also I've had `jsc` aliased to that in my personal computers for as long as I can remember)